samples/cpp/text_generation/benchmark_genai.cpp

```diff
@@ -30,7 +32,23 @@ int main(int argc, char* argv[]) try {
         return EXIT_SUCCESS;
     }
 
-    std::string prompt = result["prompt"].as<std::string>();
+    std::string prompt;
+    if (result.count("prompt") && result.count("prompt_file")) {
+        std::cout << "Prompt and prompt file should not exist together!" << std::endl;
+        return EXIT_FAILURE;
+    } else {
+        if (result.count("prompt")) {
+            prompt = result["prompt"].as<std::string>();
+        }
+        else if (result.count("prompt_file")) {
+            prompt = utils::read_prompt(result["prompt_file"].as<std::string>());
+        }
+        else {
+            prompt = "The Sky is blue because";
+            std::cout << "Run with default prompt:" << prompt << std::endl;
```

Comment:
The old default was more readable because it's part of cmd args: `("p,prompt", "Prompt", cxxopts::value<std::string>()->default_value("The Sky is blue because"))`.

Can you revert the default?